static/src/js/util/retrievals.js

```diff
@@ -108,12 +108,31 @@ export const prepareRetrievalParams = (state) => {
 
     const params = buildGranuleSearchParams(preparedParams)
 
+    const allVariables = accessMethods[selectedAccessMethod].variables
+
     returnValue.granule_params = params
     returnValue.access_method = pick(
       accessMethods[selectedAccessMethod],
       permittedAccessMethodFields
     )
 
+    if (returnValue.access_method.selectedVariables) {
```

Comment:
avoid chaining multiple calls into nested objects like this, and below in the loop